packages/expo-notifications/android/src/main/java/expo/modules/notifications/serverregistration/InstallationId.java

```diff
@@ -25,6 +25,7 @@
 
   public static final String LEGACY_UUID_KEY = "uuid";
   public static final String UUID_FILE_NAME = "expo_installation_uuid.txt";
+  public static final String SCOPED_UUID_FILE_NAME = "expo_notifications_installation_uuid.txt";
```

Comment:
In the other PR this was called expo_notifications_installation_**id**.txt -- is that going to cause problems?